CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Removed `Ext4Error::as_corrupt`.
+* Removed `impl From<Corrupt> for Ext4Error`.
```

Comment:
nit: Why no mention of removing `impl From<CorruptKind> for Corrupt`?

Looks like they're both public...

Reply:
`CorruptKind` is `pub(crate)`, not public. So this impl is not visible to users.